src/client/Microsoft.Identity.Client/TelemetryCore/Internal/HttpTelemetryContent.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.TelemetryCore.Internal.Constants;
+using Microsoft.Identity.Client.TelemetryCore.Internal.Events;
+
+namespace Microsoft.Identity.Client.TelemetryCore.Internal
+{
+    internal class HttpTelemetryContent
+    {
+        public HttpTelemetryContent(EventBase evt)
+        {
+            if (evt != null)
+            {
+                evt.TryGetValue(MsalTelemetryBlobEventNames.ApiIdConstStrKey, out string apiId);
+                evt.TryGetValue(MsalTelemetryBlobEventNames.MsalCorrelationIdConstStrKey, out string correlationId);
+                evt.TryGetValue(MsalTelemetryBlobEventNames.ApiErrorCodeConstStrKey, out string errorCode);
+                evt.TryGetValue(MsalTelemetryBlobEventNames.ForceRefreshId, out string forceRefresh);
+
+                ApiId = apiId ?? string.Empty;
+                CorrelationId = correlationId ?? string.Empty;
+                LastErrorCode = errorCode ?? string.Empty;
+                ForceRefresh = forceRefresh ?? string.Empty;
+            }
+        }
+
+        public string LastErrorCode { get; set; } = string.Empty;
+        public string ApiId { get; set; } = string.Empty;
+        public string CorrelationId { get; set; } = string.Empty;
+        public string ForceRefresh { get; set; } = string.Empty;
+
+        public string GetCsvAsPrevious()
+        {
+            if (string.IsNullOrWhiteSpace(ApiId))
+            {
+                return string.Empty;
+            }
+
+            // csv expected format:
+            // 1|api_id,correlation_id,last_error_code|platform_config
+            string[] myValues = new string[] {
```

Comment:
try to refer to local variables without the pronoun `my`. Instead, give a name that has meaning in the local context, for example `dataToCollect`